### ISSUE TITLE
Question about mask Observation

### ISSUE THREAD

**Issue by XueyangFeng** [anchen1011/FireAct#5]
Hello, thanks for your great work!
I see that you try to mask the Observation line in your lora fine-tune code, but why you not use the masked label ? I think this line maybe "result["labels"] =ids"
"https://github.com/anchen1011/FireAct/blob/94c6189f04b10fccae6bcd97a4632d6119266899/finetune/llama_lora/finetune.py#L146
Looking forward to your reply!
